app/scripts/PixiTrack.js

```diff
@@ -108,11 +108,28 @@ function getWidthBasedResolutionText(
  * @property {string=} dataTransform
  */
 
-/** @extends {Track<PixiTrackOptions>} */
+/**
+ * @typedef {import('./Track').ExtendedTrackContext<{ scene: import('pixi.js').Container}>} PixiTrackContext
+ */
+
+/**
+ * @template T
+ * @typedef {T & PixiTrackContext} ExtendedPixiContext
+ */
+
+/**
+ * @template T
+ * @typedef {T & PixiTrackOptions} ExtendedPixiOptions
+ */
+
+
+/** 
+ * @template {ExtendedPixiOptions<{[key: string]: any}>} Options
```

Comment:
Since classes which extend this one call `super` with the `options` which later gets set to `this.options`, I need a template of an object which must have all of the PixiTrackOptions but can have any other additional property